src/main/java/ApplyToGroup/ApplyToGroupResponseModel.java

```diff
@@ -0,0 +1,31 @@
+package ApplyToGroup;
+
+import Entities.Group;
+import Entities.User;
+
+public class ApplyToGroupResponseModel {
+
+    private User user;
+    private Group group;
+
+    public ApplyToGroupResponseModel(User user, Group group) {
+
+        this.user = user;
+        this.group = group;
+    //}
```

Comment:
Just wondering, why are you passing user in the response model?

Reply:
Yes, I don't think we should be passing around entities (even if they're now interfaces), right? Especially in the response model, as the view only needs the username string and the groupname string or any other basic descriptors.

Reply:
Yeah I also noticed this after we decided to work with usernames instead of userIDs. I made the necessary changes and they will be reflected in my new pull request.